scripts/ios_shortcut_input.py

```diff
@@ -0,0 +1,28 @@
+#
+# ios_shortcut_input.py
```

Comment:
Can we rename this to `list_sites.py` or something? :)

Reply:
Hmm, `gimme_those_site_keys.py`?

Reply:
`gimme_dem`